packages/dds/task-manager/src/index.ts

```diff
@@ -3,5 +3,11 @@
  * Licensed under the MIT License.
  */
 
+/**
+ * Contains a distributed data structure for queueing and processing tasks that must be completed by a single client.
```

Comment:
@scottn12 interested in your thoughts on a top-level summary for the package. My summary is probably bad, so any feedback would be great. Also example scenarios where a user may want to use this?

I'll also transcribe these into the README.